webservice/pom.xml

```diff
@@ -173,6 +173,16 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why is it added in webservice module ? It should be part of the storage-tools module.